projects/afterme.md

```diff
@@ -0,0 +1,54 @@
+## Project Name <!-- Add your project name here with format "Project Name"-->
+After Me
+## Category 
+<!--developer tooling, application, wallet, infrastructure, etc-->
+Application
+## Project Description
+<!--Describe your project in a few sentences. -->
+Trustless estate planning solution that allows users to manage their digital assets in life and beyond.
+## Project Status
+<!--brainstorming, fundraising, under development, beta, shipped, etc-->
+MVP operational
+## Previews
+<!--Add some screenshots to give a preview of your product-->
+https://i.ibb.co/BzM0kTZ/1-set-up-contract.jpg
+https://i.ibb.co/V3mCY6K/manage-contract.jpg
+https://i.ibb.co/097dmHw/my-content.jpg
+https://i.ibb.co/T8MS1wG/archives.jpg
+
+## Target Audience
+<!--Describe who will be your project's users-->
+Customer segments:<ol>
+  <li>Users of traditional centralized estate planning solutions that are motivated by After Me’s lower price point </li>
+
+<li>Post-mortem vloggers that wish to leave behind a record of their personality without being exposed to public scrutiny</li>
+
+<li>Public persons interested in retaining ownership of their natural identity through a claiming service (value prop. on the product roadmap)</li>
+
+
+## Github repo
+<!--Attach a link to your GitHub repo if it's OSS-->
+https://github.com/EightRice/afterme
+
+## Website
+<!--Link your website if available-->
+https://www.deathisreal.com Password: cl_hack **ONLY ON KOVAN TESTNET**
+
+## Docs
+<!--Including a link to your project docs!-->
+https://drive.google.com/file/d/1-fTdNrganmovKyCGh5x71nsg9SiDe2Ir/view?usp=sharing
+
+## Team Info
+<!-- Introduce your amazing team - how many team members are working on this project and who are they?-->
+After Me is currently a one-man-show. Andrei Taranu - product manager www.andreitaranu.ml
+
+## How the community can engage
+GitHub Discussion: <!--Start a disucssion with the community here: https://gastmobronsita@gmail.comithub.com/filecoin-project/community/discussions/new and attach the link!-->  
+Email: astmobronsita@gmail.com 
+Twitter:  @andreitaranu
+Discord:  https://discord.gg/RKTHJZF
+
+## How to Contribute
+https://github.com/ipfs/community/discussions/485
```

Comment:
Can you add the discussion link up above?

Reply:
I added the link to the project repo's README.md file. Is that what you meant?